libs/styles/styles.css

```diff
@@ -550,6 +553,8 @@ span.icon.margin-right { margin-right: 8px; }
 
 span.icon.margin-left { margin-left: 8px; }
 
+span.icon.margin-inline-start { margin-inline-start: 8px; }
```

Comment:
This may have already been contributed in https://github.com/adobecom/milo/blob/stage/libs/styles/styles.css#L562

Reply:
oops yep, your right. Thanks